style.css

```diff
@@ -441,8 +441,8 @@ li.social-icon {
 @media (min-width: 768px) {
   .contact-section-bg {
     background-image: url(../images/background/contact-bg-lg.svg);
-    background-position: center center;
     background-size: 1290px 718px;
+    background-position: center center;
```

Comment:
- [ ] nice work. i can see you are using two media queries for your desktop version. Kindly use one and implement all the changes under. Nice work